topics/dev/tutorials/visualization-charts/tutorial.md

```diff
@@ -316,6 +316,14 @@ instance, just make sure to properly refresh your browser.
 
 Lets test this.
 
+> <comment-title></comment-title>
```

Comment:
```suggestion
> <comment-title>Clearing the Cache</comment-title>
```

if you want to provide a nicer title or so.

Also for the subsequent lines could you ensure each line starts with `> ` until the `{: .comment}` line which does not need a block quote